builtin/pack/builder.go

```diff
@@ -57,7 +69,7 @@ type BuilderConfig struct {
 	ProcessType string `hcl:"process_type,optional" default:"web"`
 }
 
-const DefaultBuilder = "heroku/buildpacks:18"
+const DefaultBuilder = "heroku/buildpacks:20"
```

Comment:
Should I have done this? ..... 🤔

Reply:
I think I do want to up this because :18 is going to fade out of maintenance before too long, better to upgrade to :20 as the default now.